### PR TITLE
Lisa the Hopeful Port

### PR DESCRIPTION
**Pull request by Tlaggypyro**
# New Port for {Lisa The Hopeful}

## Game Information
- **Title**: { Lisa The Hopeful }
- **URL**: { (https://taco-salad.itch.io/lisa-the-hopeful) }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/lisathehopeful/port.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "LISA The Hopeful.sh",
+        "lisathehopeful"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "LISA: The Hopeful",
+        "porter": [
+            "TLaggyPyro"
+        ],
+        "desc":Lisa The Hopeful is a fan game set  the Lisa universe",
```
Comment (mhoogcarspel) [inline ports/lisathehopeful/port.json:14]: missing a double quote there

**Review by Cebion — APPROVED** [review]

**Review by Slayer366 — APPROVED** [review]

File: ports/lisathehopeful/port.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "LISA The Hopeful.sh",
+        "lisathehopeful"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "LISA: The Hopeful",
+        "porter": [
+            "TLaggyPyro"
+        ],
+        "desc":"Lisa The Hopeful is a fan game set  the Lisa universe",
```
Comment (kloptops) [inline ports/lisathehopeful/port.json:14]: set in the Lisa universe?

**Review by kloptops — COMMENTED** [review]
Looks good, the port.json has a small typo in the description. 😄

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**